e2e_test/ddl/drop/drop_creating_mv.slt

```diff
@@ -0,0 +1,76 @@
+statement ok
+create table t(v1 int);
+
+statement ok
+insert into t select * from generate_series(1, 10000);
+
+statement ok
+flush;
+
+statement ok
+set streaming_rate_limit=1;
+
+############## Test drop foreground mv
+onlyif can-use-recover
+system ok
+psql -h localhost -p 4566 -d dev -U root -c 'create materialized view m1 as select * from t;' &
```

Comment:
This doesn't work under parallelism execution since the database name won't be `dev` anymore. 😕

Reply:
Parallel execution won't run this test. Because trigger recovery for one parallelism can affect execution of another parallelism.